<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1124</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1124</width>
    <height>664</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 237);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 253, 255);</string>
   </property>
   <widget class="QPushButton" name="category1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="category2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="category4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="category3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="category5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="category6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item:selected { background: rgb(226, 226, 226); } QMenuBar::item:pressed {  background: rgb(226, 226, 226); } QMenuBar {background-color: rgb(255, 255, 255);}
</string>
   </property>
   <widget class="QMenu" name="menuCounting_Brain_Cells">
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenuBar::item:selected { background: rgb(226, 226, 226); } QMenuBar::item:pressed {  background: rgb(226, 226, 226); } QMenuBar {background-color: rgb(255, 255, 255);}</string>
    </property>
    <property name="title">
     <string>Counting Brain Cells</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionWebsite"/>
    <addaction name="actionGithub"/>
    <addaction name="actionCredits"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions_2">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSound"/>
   </widget>
   <addaction name="menuCounting_Brain_Cells"/>
   <addaction name="menuOptions_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>View on Github</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
